content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-rabbitmq.md

```diff
@@ -25,11 +25,20 @@ This provides a foundation to understand the steps that Data Flow will automate
 
 ## Development
 
+The following sections describe how to build this stream from scratch. If you prefer, you can download a zip file containing the sources for the sample stream, unzip it, and proceed to the [build](#building) step.
```

Comment:
the build step is just for the source.  I've changed it to reference how to build the full solution here and then referencing to head over to one of the deployment sections.